packages/cardano-services/src/StakePoolSearch/DbSyncStakePoolSearchProvider/mappers.ts

```diff
@@ -60,7 +64,8 @@ export const toCoreStakePool = ({
       hexId: poolData.hexId,
       id: poolData.id,
       margin: poolData.margin,
-      metrics: {} as Cardano.StakePoolMetrics, // TODO: fetch metrics
+      metrics:
+        poolMetrics.find((metrics) => metrics.hashId === poolData.hashId)?.metrics || ({} as Cardano.StakePoolMetrics),
```

Comment:
Aren't these supposed to correspond to the same index?

Reply:
poolMetrics contains the metrics of all the fetched pools in order to only make one query to fetch all pools metrics. Then, they are matched through hashId